src/styles/variables.scss

```
@@ -0,0 +1,6 @@
+$card-color-w: rgba(255, 255, 255, 1);
+$border-color: rgba(243, 243, 243, 1);
+$font-color-db: rgba(6, 11, 53, 1);
+$font-color-gr: rgba(97, 96, 112, 1);
+$card-color-aqua: rgba(0, 172, 220, 1);
+$bodyFont: 'Roboto', sans-serif;
```

It is better to name variables more universally and try not to use abbreviations.
```suggestion
$card-color: rgba(255, 255, 255, 1);
$border-color: rgba(243, 243, 243, 1);
$title-color: rgba(6, 11, 53, 1);
$description-color: rgba(97, 96, 112, 1);
$card-color-accent: rgba(0, 172, 220, 1);
$bodyFont: 'Roboto', sans-serif;
```